Projects/Payment/Sources/Screens/PaymentsView.swift

```diff
@@ -16,45 +18,53 @@ public struct PaymentsView: View {
     }
 
     public var body: some View {
-        LoadingStoreViewWithContent(
-            PaymentStore.self,
-            [.getPaymentData],
-            [.load, .fetchPaymentStatus]
-        ) {
-            hForm {
-                VStack(spacing: 8) {
-                    upcomingPayment
-                    PresentableStoreLens(
-                        PaymentStore.self,
-                        getter: { state in
-                            state.paymentStatusData
+        successView.loading($vm.viewState)
+            .loadingWithButtonLoading($vm.viewState)
```

Comment:
We don't need both:
loading
loadingWithButtonLoading
just loadingWithButtonLoading here